app/helpers/is-access-level-in-effect.js

```diff
@@ -0,0 +1,82 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+import { isPresent } from '@ember/utils';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class IsAccessLevelInEffect extends Helper {
+  @service agendaService;
+
```

Comment:
This helper method is very specifically tuned to pieces of agendaitems.
This is not readily apparent from the helper name.

Document cards seen from subcase views won't show the same styling as document cards viewed from agendaitem views?

Reply:
Hmm, I hadn't though of this. Looking at the FP again, it only specifies rules for pieces related to agendas. From a subcase view, if a piece is linked to an agendaitem, it should still be displayed in the same way as from an agenda view (because the helper fetches all the related data and finds the linked agenda). If the piece is not also linked to an agendaitem, then it will show a draft status (unless the access level is `Intern secretarie`). I think this would be technically correct, given that Yggdrasil only propagates pieces that are linked to an agenda, right?